### ISSUE TITLE
some requests doesn't return the value

### ISSUE THREAD

**Issue by brunoqc** [slact/nchan#187]
Feel free to rename this issue's title to something better.

I'm not sure if I reported a problem like this in the past. It looks a bit like #161.

If I run twice `curl -k https://<url>`. Once I get the value and once it blocks just like there wasn't a value stored and it's waiting for a new value (it doesn't unblocks if I send a new value on the channel).

I didn't see anything in the log.

nginx 1.11.0
nchan v0.99.14

```
location ~ /sub/(\w+)$ {
        nchan_subscriber;
        nchan_message_buffer_length 1;
        nchan_channel_id $1; #first capture of the location match
    }

 location ~ /pub/(\w+)$ {
        allow <ip>;
        deny all;

        nchan_publisher;
        nchan_channel_id $1; #first capture of the location match
        nchan_message_buffer_length 1;
        nchan_message_timeout 0;
    }
```

**Comment by brunoqc** [comment]
I have `nchan_use_redis on`.

**Comment by slact** [comment]
how many workers? Did this happen just once, or are you able to reproduce it?

**Comment by brunoqc** [comment]
2 workers. I didn't try to reproduce it yet. It was a production server so I downgraded nginx to 1.9.15 and it seems fine.

I'll try to reproduce.

**Comment by slact** [comment]
Thanks, I'll investigate.

**Comment by slact** [comment]
Please update this ticket if this is still happening, I believe this issue was fixed in 0.99.15

**Comment by brunoqc** [comment]
The problem seems to be gone with 0.99.15.
